util/security/secret.py

```diff
@@ -25,5 +25,10 @@ def convert_secret_key(config_secret_key):
         secret_key = str(bytearray(list(map(ord, config_secret_key))))
 
     # Otherwise, use the bytes directly.
-    assert len(secret_key)
-    return "".join(itertools.islice(itertools.cycle(secret_key), 32))
+    assert len(secret_key) > 0
+    print(secret_key)
```

Comment:
Not sure if this was intentional but we should probably pull this `print()` statement out. I assume it was there for debugging purposes.